testdata/input/questionnaire_borgScale10.csv

```diff
@@ -1,9 +1,9 @@
 ﻿itemID;answer
-1;0;

```

Comment:
@aremazeilles  we followed your instructions to create this input file. Currently, it is generated by the TedSR software. We can manually alter the data to conform to your requirements.

Reply:
It's been some time since I made this, so I cannot provide the context, bu I guess there was some issues with having a final `;` which may be interpreted as a new column, but there is no.